tron/core/actionrun.py

```diff
@@ -324,6 +327,13 @@ def from_state(
 
     def start(self):
         """Start this ActionRun."""
+        if self.in_delay is not None:
```

Comment:
I think this part should go to start_after_delay(). Any reason to put them here instead?

Reply:
we don't want `start` to be called when we're in delay, it could happen from other events.